wd_helper.au3

```diff
@@ -347,6 +348,8 @@ Func _WD_WaitElement($sSession, $sStrategy, $sSelector, $iDelay = Default, $iTim
 			$sElement = _WD_FindElement($sSession, $sStrategy, $sSelector)
 			$iErr = @error
 
+			If $_WD_DEBUG_Saved < $_WD_DEBUG_Full Then $_WD_DEBUG = $_WD_DEBUG_None ; show only once
```

Comment:
Two questions / thoughts about this --

- Why did you choose `$_WD_DEBUG_None` over `$_WD_DEBUG_Error`?
- I wonder if showing the output from `_WD_FindElement` a single time is beneficial here. I believe it could be misleading or lead to confusion.

Reply:
I must think about.

Reply:
> >   Why did you choose $_WD_DEBUG_None over $_WD_DEBUG_Error?

> to show only once

That didn't answer my question. Let me restate it so that there's no confusion --

Why did you decide to use $_WD_DEBUG_None instead of $_WD_DEBUG_Error?

Reply:
> Why did you decide to use $_WD_DEBUG_None instead of $_WD_DEBUG_Error?

because then the errors would be shown many times, and what we are trying to achieve here is to minimize the amount of information in the logs.

But I have some idea to present.
Wait (maybe until tomorrow, maybe today) for another amendment.
Then I will speak more broadly.

Reply:
Take a look on 
[_WD_FindElement() $bCheckNoMatch Error handling](https://github.com/Danp2/au3WebDriver/pull/290/commits/b719bfd9f61303ea6293015aa6652cd37507fdd5)
[b719bfd](https://github.com/Danp2/au3WebDriver/pull/290/commits/b719bfd9f61303ea6293015aa6652cd37507fdd5)

and fixed version:
[_WD_WaitElement() should exits on error](https://github.com/Danp2/au3WebDriver/pull/290/commits/2a6080d95dfa049acf31fd1bbfb1b91301376625)


or simply here:
https://github.com/Danp2/au3WebDriver/pull/290/files/736cd808aa0b764d89ee01c6677718eb0f6ededc..2a6080d95dfa049acf31fd1bbfb1b91301376625

Reply:
I think that when executing a function is in loop and waiting for Match/NoMatch, when some other fatal error happens for example browser EXE or WebDriver EXE is closed durring, or when user pass invalid XPath/CSSSelector then there is no reason to wait.

So I added this:
```autoit
If $iErr <> $_WD_ERROR_Success And $iErr <> $_WD_ERROR_NoMatch Then
	ExitLoop
```
to exit loop on such scenario

Reply:
This is my answer to this:

> > * I wonder if showing the output from `_WD_FindElement` a single time is beneficial here. I believe it could be misleading or lead to confusion.
> 
> I must think about. Give me some time. I plan to have a non computer related sunday.


and to this:


> > Why did you decide to use $_WD_DEBUG_None instead of $_WD_DEBUG_Error?
> 
> because then the errors would be shown many times, and what we are trying to achieve here is to minimize the amount of information in the logs.
> 
> But I have some idea to present. Wait (maybe until tomorrow, maybe today) for another amendment. Then I will speak more broadly.

Reply:
Finally I get it here:
[prevent multiple errors from _WD_FindElement()](https://github.com/Danp2/au3WebDriver/pull/290/commits/b1cfd23e2394ac23da18486da682db417b2d17d1)

Reply:
and here:
[_WD_WaitElement - $_WD_DEBUG_Full and $sParameters](https://github.com/Danp2/au3WebDriver/pull/290/commits/7d532fac743c55fe07a0057f26dbdd5ede6ceee6)

Reply:
For me all is fine here. Do you still waiting on some information/change from me ?